README.md

```diff
@@ -22,7 +22,7 @@ Please note: The code in these repos is sourced from the DataRobot user communit
 | Title | Accelerator Topic | What it's good for | 3rd Party Integration | Extensibility to other Integrations |
 |---|---|---|---|---|
 | [Automated Feature Discovery with Multiple Tables](https://github.com/datarobot-community/ai-accelerators/tree/main/advanced-experimentation/AFD) | Automated Feature Discovery workflow | Repeatable end-to-end workflow for automated time-aware feature engineering in Snowflake from data to batch predictions | Snowflake | Horizontal approach that applies to classification/ regression problems with a time-component; applies to all datasources  |
-| [Azure Storage End-to-End Workflow with DataRobot](https://github.com/datarobot-community/ai-accelerators/blob/main/end-to-end/Azure_End_to_End.ipynb) | Azure End-to-End | Repeatable end-to-end workflow in Azure from data to batch predictions | Azure | Low, Azure focused |
+| [Azure Storage End-to-End Workflow with DataRobot](https://github.com/datarobot-community/ai-accelerators/blob/main/end-to-end/Azure_End_to_End) | Azure End-to-End | Repeatable end-to-end workflow in Azure from data to batch predictions | Azure | Low, Azure focused |
```

Comment:
```suggestion
| [Azure Storage End-to-End Workflow with DataRobot](https://github.com/datarobot-community/ai-accelerators/blob/main/end-to-end/Azure_End_to_End/Azure_End_to_End.ipynb) | Azure End-to-End | Repeatable end-to-end workflow in Azure from data to batch predictions | Azure | Low, Azure focused |
```

Sorry, I comment on the commit and not the PR itself. Let's keep the link going to the notebook, but update the link to reflect the structure.